prime-router/metadata/fhir_mapping/hl7/resource/ServiceRequest.yml

```diff
@@ -290,45 +300,73 @@ extension_9:
         system_code: "v2-0123"
 
 extension_10:
+  condition: $code NOT_EQUALS S && $code NOT_EQUALS A && $code NOT_EQUALS R && $code NOT_EQUALS U
   generateList: true
   expressionType: nested
+  vars:
+    code: String, TQ1.9.1
   expressionsMap:
     url:
       type: SYSTEM_URL
-      value: start-of-service
-    valueDateTime:
-      type: STRING
-      valueOf: "GeneralUtils.dateTimeWithZoneId(dateTimeIn,ZONEID)"
-      expressionType: JEXL
+      value: service-priority
+    valueCodeableConcept:
+      valueOf: datatype/CodeableConcept_var
+      expressionType: resource
       vars:
-        dateTimeIn: TQ1.7
+        system: SYSTEM_URL, $system_code
+      constants:
+        system_code: "v2-0485"
 
 extension_11:
   generateList: true
   expressionType: nested
   expressionsMap:
     url:
       type: SYSTEM_URL
-      value: end-of-service
+      value: order-effective-date
     valueDateTime:
-      type: STRING
-      valueOf: "GeneralUtils.dateTimeWithZoneId(dateTimeIn,ZONEID)"
-      expressionType: JEXL
-      vars:
-        dateTimeIn: TQ1.8
+        type: STRING
+        valueOf: "GeneralUtils.dateTimeWithZoneId(dateTimeIn,ZONEID)"
+        expressionType: JEXL
+        vars:
+          dateTimeIn: ORC.15
```

Comment:
Is this one reflected in our sharepoint spreadsheet for ORC? I'm a bit confused by OCR.15 being here twice.

Reply:
I updated the spreadsheet, but it might not be very clear why we have the value here twice. There's no mapping for ORC.15 yet, so we had to add an extension for it. ORC.15 was being mapped to occurrence  before, but the new mappings for TQ1 use that. We thought it would be a good idea to keep the mapping for ORC.15 to occurrence for whenever the TQ1 values are null.

Something that was confusing to me - in case you haven't seen it before - is that even though occurrenceDateTime and occurrenceTiming look like different fields they're actually the same resource and you can only have one of them being mapped to occurrence. That's why occurrenceDateTime needs the condition.

Reply:
Ah okay, I think I just overlooked the fact that ORC-15 has two rows in the spreadsheet, so this all makes sense, thanks for the explanation!